### PR TITLE
Feature/#14 home page with blog posts

### PR DESCRIPTION
**Pull request by SgiobairOg**
Multiple commits to implement the PostList and PostListItem components as well as implementing styles and tweaking the layout.

### CODE REVIEW

File: packages/blog-with-markdown-files/contents/blog-posts/how-to-get-better-with-tailwind.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: How to Get Better with Tailwind
+slug: how.toget.better.with.tailwind
+publishDate: '09-06-2020'
+author: Jason Wilson
+published: true
+featuredImage: 'http://picsum.photos/seed/You/800/500'
+---
+
+You can't. You're old. Give up. Pellentesque condimentum velit vel justo rutrum, sit amet commodo diam tincidunt. Nunc diam massa, interdum ut aliquet at, scelerisque ac ex. Integer cursus sem ac pretium posuere. Ut at odio nulla. Phasellus nec ante luctus, egestas dui id, maximus dui. In aliquam elit sit amet sollicitudin luctus. Nunc nec leo quis ante vestibulum egestas. In dignissim libero vitae congue bibendum. Sed iaculis eros a leo pellentesque, et ultrices leo malesuada. Nullam ultrices rutrum accumsan. Pellentesque tempus sapien et vestibulum placerat.
```
[the remark this answers is not in the document]
Comment (lorenzogm) [reply packages/blog-with-markdown-files/contents/blog-posts/how-to-get-better-with-tailwind.md:10]: 🤣

**Comment by lorenzogm** [comment]
Waiting for the changes from last FE PA meeting to have a look ;) @SgiobairOg

File: packages/blog-with-markdown-files/components/organisms/PostListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <PostListItem_
+      className={`post-list__post-item ${hasFeaturedImage ? 'post-list__post-item--image' : ''}`}
+      style={backgroundStyles}
+    >
+      <PostDetailsWrapper_
+        className={`post-item__details-wrapper ${hasFeaturedImage ? 'post-item__details-wrapper--image' : ''}`}
+      >
+        <PostDetails_ className={`post-item__details`}>
+          <PostCategories_ className="post-item__categories">Javascript</PostCategories_>
```
[the remark this answers is not in the document]
Comment (SgiobairOg) [reply packages/blog-with-markdown-files/components/organisms/PostListItem.tsx]: Took care of this too while I was in there.

File: packages/blog-with-markdown-files/components/organisms/PostList/PostList.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import styled from '@emotion/styled'
+import tw from '@tailwindcssinjs/macro'
+import { BlogPost } from 'types/blog'
+
+import PostListItem from 'components/organisms/PostListItem/PostListItem'
+
+const PostList_ = styled.ul(tw`flex w-full flex-wrap justify-between items-center`)
```
Comment (lorenzogm) [inline packages/blog-with-markdown-files/components/organisms/PostList/PostList.tsx:7]: I think we should move the styling for the first 5th post here, using a prop

File: packages/blog-with-markdown-files/components/organisms/PostList/PostList.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+type PostListProps = {
+  blogPosts: BlogPost[]
+}
+
+export default function PostList({ blogPosts }: PostListProps): React.ReactElement {
+  return (
+    <PostList_>
+      {blogPosts.map((post) => {
+        return <PostListItem blogPost={post} key={post.id}></PostListItem>
```
Comment (lorenzogm) [inline packages/blog-with-markdown-files/components/organisms/PostList/PostList.tsx:17]: here we can pass a prop "position={index}" to add the border with the right color

File: packages/blog-with-markdown-files/components/organisms/PostListItem/PostListItem.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+export default function PostListItem({ blogPost }: PostListItemProps): React.ReactElement {
+  return (
+    <PostListItem_ featuredImage={blogPost.featuredImage}>
+      <PostDetailsWrapper featuredImage={blogPost.featuredImage}>
+        <PostDetails>
+          <PostCategories>{blogPost.category}</PostCategories>
+          <PostTitle>{blogPost.title}</PostTitle>
+          <PostByline>
+            {blogPost.author} | {blogPost.publishDate}
```
Comment (lorenzogm) [inline packages/blog-with-markdown-files/components/organisms/PostListItem/PostListItem.tsx:25]: I think that using template literals is better when writing text to handle the spaces: `${blogPost.author} | {blogPost.publishDate}`

File: packages/blog-with-markdown-files/types/blog.d.ts [hunk 6]
```diff
@@ -7,4 +7,5 @@ export type BlogPost = {
   published: boolean
   publishDate: string
   featuredImage?: string
+  category?: string
```
Comment (lorenzogm) [inline packages/blog-with-markdown-files/types/blog.d.ts:10]: Maybe we can have `tags: string[]`, instead of a category

**Review by lorenzogm — APPROVED** [review]
LGTM, but there was something with the indentation, isn't it?

**Comment by SgiobairOg** [comment]
Yeah there is but since it wanted to change every file I thought it was a bit much. Now that we've looked though I'll run it and push up the formatting then merge

**Review by SgiobairOg — COMMENTED** [review]
Linter changes only here

**Pull request merged**